client-side/src/components/statistics/graphBar.jsx

```diff
@@ -0,0 +1,34 @@
+import * as React from 'react';
+import { useQuery } from '@apollo/client';
+import { BarChart } from '@mui/x-charts/BarChart';
+import { useAppSelector } from '../../redux/store.jsx';
+import Loader from '../../stories/loader/loader.jsx'
+import { getVisitedWebsitesByDate, formatDate } from './graphsUtils.js';
+import { GET_USERS } from './queries.js';
+
+export default function GraphBar({ startDate, endDate }) {
+  const users = useQuery(GET_USERS);
+  const user = useAppSelector((state) => state.user.currentUser);
+  startDate = formatDate(new Date(startDate.$d));
+  endDate = formatDate(new Date(endDate.$d));
+  const result1 = getVisitedWebsitesByDate(users, user, startDate);
+  const result2 = getVisitedWebsitesByDate(users, user, endDate);

```

Comment:
please rename to more meaningful name

Reply:
I changed it to dataOfEndDate and dataOfStartDate.